02-Authentication-HS256/config/application.rb

```diff
@@ -8,7 +8,7 @@
 require "action_mailer/railtie"
 require "action_view/railtie"
 require "action_cable/engine"
-require "sprockets/railtie"
+# require "sprockets/railtie"
 require "rails/test_unit/railtie"
```

Comment:
Do we need these frameworks?

Reply:
not all of them. I reduced it to the bare minimum: 

```ruby
require "rails"
# Pick the frameworks you want:
require "action_controller/railtie"
require "action_mailer/railtie"
```